tasks/copy_to_archive_adapter/README.md

```diff
@@ -0,0 +1,292 @@
+
+The output of this lambda is a dictionary with a `granules` and `copied_to_orca` attributes.  See the schema [output file](https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_archive_adapter/schemas/output.json) for more information. Below is an example of the output:
+
+```json
+{
+	"granules": [
+    {
+      "granuleId": "MOD09GQ.A2017025.h21v00.006.2017034065109",
+      "dataType": "MOD09GQ",
+      "version": "006",
+      "createdAt": 628021800000,
+      "files": [
+        {
+          "name": "MOD09GQ.A2017025.h21v00.006.2017034065109.hdf",
+          "path": "MOD09GQ/006",
+          "size": 6,
+          "time": 1608318366000,
+          "bucket": "orca-sandbox-internal",
+          "url_path": "MOD09GQ/006/",
+          "type": "",
+          "filename": "s3://orca-sandbox-internal/file-staging/orca-sandbox/MOD09GQ___006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf",
+          "fileStagingDir": "file-staging/orca-sandbox/MOD09GQ___006"
+        },
+        {
+          "name": "MOD09GQ.A2017025.h21v00.006.2017034065109.hdf.met",
+          "path": "MOD09GQ/006",
+          "size": 6,
+          "time": 1608318366000,
+          "bucket": "orca-sandbox-internal",
+          "url_path": "MOD09GQ/006",
+          "type": "",
+          "filename": "s3://orca-sandbox-internal/file-staging/orca-sandbox/MOD09GQ___006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf.met",
+          "fileStagingDir": "file-staging/orca-sandbox/MOD09GQ___006"
+        },
+        {
+          "name": "MOD09GQ.A2017025.h21v00.006.2017034065109_ndvi.jpg",
+          "path": "MOD09GQ/006",
+          "size": 6,
+          "time": 1608318372000,
+          "bucket": "orca-sandbox-internal",
+          "url_path": "MOD09GQ/006",
+          "type": "",
+          "filename": "s3://orca-sandbox-internal/file-staging/orca-sandbox/MOD09GQ___006/MOD09GQ.A2017025.h21v00.006.2017034065109_ndvi.jpg",
+          "fileStagingDir": "file-staging/orca-sandbox/MOD09GQ___006"
+        }
+      ],
+      "sync_granule_duration": 1676
+    }
+  ],
+	"copied_to_orca": [
+      "s3://orca-sandbox-protected/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf",
+      "s3://orca-sandbox-private/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf.met",
+      "s3://orca-sandbox-public/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109_ndvi.jpg",
+      "s3://orca-sandbox-private/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.cmr.xml"
+  ]
+}
+```
+
+## Configuration
+
+As part of the [Cumulus Message Adapter configuration](https://nasa.github.io/cumulus/docs/workflows/input_output#cma-configuration) 
+for `copy_to_archive`, the `excludedFileExtensions`, `s3MultipartChunksizeMb`, `providerId`, `executionId`, `collectionShortname`, `collectionVersion`, `defaultBucketOverride`, and `defaultStorageClassOverride` keys must be present under the 
+`task_config` object as seen below. 
```

Comment:
According to your copy_to_archive README, it says:
```
`excludedFileExtensions`, `s3MultipartChunksizeMb`, `providerName`, `defaultBucketOverride`, and `defaultStorageClassOverride` are stored as paths, and will be retrieved in-code.
```

Reply:
The section in the adapter is referring to the workflow, but my title didn't reflect that. Updated, and clarified some wording in the copy_to_archive readme.